ui/src/main/java/ucar/nc2/ui/simplegeom/SimpleGeomTable.java

```diff
@@ -0,0 +1,859 @@
+
+    public String getZ() {
+      return z;
+    }
+
+    public String getT() {
+      return t;
+    }
+
+    public String getEns() {
+      return ens;
+    }
+
+    public String getRt() {
+      return rt;
+    }
+
+    public String getShape() {
+      return dims;
+    }
+
+    public void setShape(String dims) {
+      this.dims = dims;
+    }
+
+    private String getAxisName(CoordinateAxis axis) {
+      if (axis != null)
+        return (axis.isCoordinateVariable()) ? axis.getShortName() : axis.getNameAndDimensions(false);
+      return "";
+    }
+  }
+
+  public class GeoCoordinateSystemBean {
+    private GridCoordSystem gcs;
+    private String proj, coordTrans;
+    private int ngrids = -1;
+
+    // no-arg constructor
+    public GeoCoordinateSystemBean() {
+    }
+
+    public GeoCoordinateSystemBean(GridDataset.Gridset gset) {
+      gcs = gset.getGeoCoordSystem();
+
+      setNGrids(gset.getGrids().size());
+
+      ProjectionImpl proj = gcs.getProjection();
+      if (proj != null)
+        setProjection(proj.getClassName());
+
+      int count = 0;
+      StringBuffer buff = new StringBuffer();
+      List ctList = gcs.getCoordinateTransforms();
+      for (int i = 0; i < ctList.size(); i++) {
+        CoordinateTransform ct = (CoordinateTransform) ctList.get(i);
+        if (count > 0) buff.append("; ");
+        // buff.append( ct.getTransformType());
+        if (ct instanceof VerticalCT) {
+          buff.append(((VerticalCT) ct).getVerticalTransformType());
+          count++;
+        }
+        if (ct instanceof ProjectionCT) {
+          ProjectionCT pct = (ProjectionCT) ct;
+          if (pct.getProjection() == null) { // only show CT if theres no projection
+            buff.append("-").append(pct.getName());
+            count++;
+          }
+        }
+      }
+      setCoordTransforms(buff.toString());
+    }
+
+    public String getName() {
+      return gcs.getName();
+    }
+
+    public boolean isRegularSpatial() {
+      return gcs.isRegularSpatial();
+    }
+
+    public boolean isLatLon() {
+      return gcs.isLatLon();
+    }
+
+    public boolean isGeoXY() {
+      return ((GridCoordSys) gcs).isGeoXY();
+    }
+
+    /* public boolean isProductSet() { return isProductSet; }
+    public void setProductSet(boolean isProductSet) { this.isProductSet = isProductSet; }  */
+
+    public int getDomainRank() {
+      return gcs.getDomain().size();
+    }
+
+    public int getRangeRank() {
+      return gcs.getCoordinateAxes().size();
+    }
+
+    public int getNGrids() {
+      return ngrids;
+    }
+
+    public void setNGrids(int ngrids) {
+      this.ngrids = ngrids;
+    }
+
+    public String getProjection() {
+      return proj;
+    }
+
+    public void setProjection(String proj) {
+      this.proj = proj;
+    }
+
+    public String getCoordTransforms() {
+      return coordTrans;
+    }
+
+    public void setCoordTransforms(String coordTrans) {
+      this.coordTrans = coordTrans;
+    }
+  }
+

```

Comment:
Ditto! Break up the bean band, or get them back together?

Reply:
same as above.